### PR TITLE
Move documentation to docs/

### PR DESCRIPTION
**Pull request by tomchop**
Once this is merged we can point dftimewolf.readthedocs.io to it

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+import re
```
Comment (Onager) [inline docs/conf.py]: Please import unicode_literals here, unless there's some reason not to do so.

File: docs/module-list.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## Collectors
+
+* `FilesystemCollector` - a simple collector that just passes a local path on to
+the processors.
+
+### GRR hunts
+
+Launch or fetch results from fleet-wide GRR hunts.
+  * `GRRHuntArtifactCollector` - Launches a fleet-wide GRR
+  `ArtifactCollectorFlow`
```
Comment (Onager) [inline docs/module-list.md]: I think you forgot to add a description here

Comment (tomchop) [reply docs/module-list.md]: I meant to write "Launches a fleet-wide GRR `ArtifactCollectorFlow`" on the same line.

File: docs/conf.py (hunk 1 above)
Comment (tomchop) [reply docs/conf.py]: Done.

**Pull request merged**